mne/tests/test_epochs.py

```diff
@@ -57,6 +59,66 @@
 rng = np.random.RandomState(42)
 
 
+def test_handle_duplicate_events():
+    """Test handling of duplicate events."""
+    # A general test case
+    EVENT_ID = {'aud': 1, 'vis': 2}
+    EVENTS = np.array([[0, 0, 1], [0, 0, 2]])
+
+    with pytest.raises(RuntimeError, match='Event time samples were not uniq'):
+        _handle_duplicate_events(EVENTS, EVENT_ID, event_repeated='error')
+
+    with pytest.raises(ValueError, match='`event_repeated` must be one of '):
+        _handle_duplicate_events(EVENTS, EVENT_ID, event_repeated='bogus')
```

Comment:
If you use `_check_option` you no longer have to test this! its already taken care :)